### PR TITLE
Move containerd node e2e config to test infra

### PR DESCRIPTION
**Pull request by Random-Liu**
Depends on https://github.com/containerd/cri/pull/789.

/hold

This PR:
1) Moves containerd node e2e configuration into test-infra.
2) Add branch test for containerd 1.1 + kubernetes 1.10.

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -82,11 +82,23 @@
       "sig-node"
     ]
   },
+  "ci-containerd-build-1-1": {
+    "args": [
+      "--env=DEPLOY_DIR=containerd/release-1.1",
```
Comment (krzyzacy) [inline jobs/config.json]: this should be in prow config

Comment (Random-Liu) [reply jobs/config.json]: Hm? Will move then :P

File: prow/config.yaml [hunk 2]
```diff
@@ -7863,7 +7910,7 @@ periodics:
       - --repo=github.com/containerd/cri=master
       - --timeout=90
       - --
-      - "--node-args=--image-config-file=test/containerd/image-config.yaml --image-config-dir=$GOPATH/src/github.com/containerd/cri"
+      - "--node-args=--image-config-file=image-config.yaml --image-config-dir=/workspace/test-infra/jobs/e2e_node/containerd/containerd-master"
```
[the remark this answers is not in the document]
Comment (Random-Liu) [reply prow/config.yaml]: The config is the same, it just moved to another place. :)

**Comment by krzyzacy** [comment]
/lgtm
/hold
you might want to add an OWNERS file to the node config subdir

**Comment by Random-Liu** [comment]
@krzyzacy Done! Thanks. :D

**Pull request merged**